### PR TITLE
lilypond: fix 62801

### PR DESCRIPTION
**Pull request by emcrisostomo**
###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 11.4 20F71 x86_64
Xcode 12.5 12E262

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

Closes: https://trac.macports.org/ticket/62801

### CODE REVIEW

**Comment by emcrisostomo** [comment]
@kencu, I requested a review from you if you don't mind

**Review by nerdling — APPROVED** [review]

**Comment by nerdling** [comment]
I'll go ahead wait for Ken before merging.

**Comment by kencu** [comment]
Thanks for asking - as it turns out, my systems are all tied up trying to get this latest clang/llvm-12 update out the door, and as it happens, none of them can run 11.3 (I use several MacPros, and OpenCore got them to 11.2.3, but can't yet get them to 11.3).

So -- it LGTM, and if it works for you fellows, go for it!

**Review by kencu — APPROVED** [review]

**Pull request merged**
